### PR TITLE
[Product Partnerships] Added AWS GameLift & AWS WorkSpaces

### PR DESCRIPTION
**Pull request by rahul188**
- [X] Did you check you NRQL syntax? - Does it work?
- [X] Did you include a Data source and Documentation reference?
- [X] Are all documentation links publicly accessible?
- [X] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [X] Did you check your descriptive content for spelling and grammar errors?
- [X] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [X] Does the PR contain a screenshot for each of your dashboards?
- [X] Do your screenshots show data?
- [X] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [X] Did you check that your alerts actually work?

### CODE REVIEW

File: quickstarts/aws/amazon-workspaces/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+level: New Relic
+authors:
+  - New Relic
+title: Amazon WorkSpaces
+documentation:
+  - name: Amazon WorkSpaces installation docs
+    description: |
+      Monitor Amazon WorkSpaces by connecting AWS to New Relic.
+    url: >-
+      https://docs.newrelic.com/docs/infrastructure/amazon-integrations/aws-integrations-list/amazon-WorkSpaces-monitoring-integration/
```
Comment (pkosanam) [inline quickstarts/aws/amazon-workspaces/config.yml:21]: Observed this documentation link is not live.

**Review by pkosanam — CHANGES_REQUESTED** [review]
Is it appropriate to create alert-policy for "amazon-gamelift" ?

File: quickstarts/aws/amazon-workspaces/config.yml (hunk 1 above)
Comment (rahul188) [reply quickstarts/aws/amazon-workspaces/config.yml]: @pkosanam I have updated the path. its working now

**Comment by rahul188** [comment]
> Is it appropriate to create an alert policy for "amazon-gamelift" ?

I have analysed the GameLift metrics but supported metrics are not that relevant to add inside alerts.

**Review by MichelLosier — APPROVED** [review]

**Comment by rahul188** [comment]
closing this PR as I have raised separate PR for both entities.

**Pull request closed without merging**